Containers/Ubuntu-22/Dockerfile

```diff
@@ -43,12 +43,17 @@ RUN apt-get update && \
     apt-get install --yes --no-install-recommends \
         bison \
         ca-certificates \
+        clang \
         curl \
+        device-tree-compiler \
         flex \
         git \
         gnupg \
         gnupg2 \
         lcov \
+        libssl-dev \

```

Comment:
Would you want to update them here? Or wait for the sync event to cover them?

Reply:
The container build workflow was able to verify the changes together. I removed the other changes that will come in through the file sync so the sync modifies the dockerfiles similarly for both Ubuntu-22 and Ubuntu-24.